ios/Capacitor/Capacitor/Plugins/SplashScreen.swift

```diff
@@ -1,149 +1,209 @@
-import Foundation
 import AudioToolbox
+import Foundation
 
 @objc(CAPSplashScreenPlugin)
-public class CAPSplashScreenPlugin : CAPPlugin {
+public class CAPSplashScreenPlugin: CAPPlugin {
   var imageView = UIImageView()
   var image: UIImage?
   var call: CAPPluginCall?
   var hideTask: Any?
   var isVisible: Bool = false
-  
+
   let launchShowDuration = 3000
   let launchAutoHide = true
-  
+
   let defaultFadeInDuration = 200
   let defaultFadeOutDuration = 200
   let defaultShowDuration = 3000
   let defaultAutoHide = true
-  
+
   public override func load() {
     buildViews()
     showOnLaunch()
   }
-  
+
   // Show the splash screen
   @objc public func show(_ call: CAPPluginCall) {
     self.call = call
-    
+
     if image == nil {
-      call.error("No image named \"Splash\" found. Please check your Assets.xcassets for a file named Splash")
+      call.error(
+        "No image named \"Splash\" found. Please check your Assets.xcassets for a file named Splash"
+      )
       return
     }
-    
+
     let showDuration = call.get("showDuration", Int.self, defaultShowDuration)!
     let fadeInDuration = call.get("fadeInDuration", Int.self, defaultFadeInDuration)!
     let fadeOutDuration = call.get("fadeOutDuration", Int.self, defaultFadeOutDuration)!
     let autoHide = call.get("autoHide", Bool.self, defaultAutoHide)!
-    
-    showSplash(showDuration: showDuration, fadeInDuration: fadeInDuration, fadeOutDuration: fadeOutDuration, autoHide: autoHide, completion: {
-      call.success()
-    }, isLaunchSplash: false)
+    let backgroundColor = call.get("backgroundColor", String.self)
```

Comment:
It should be like this:
```suggestion
    let backgroundColor = getConfigValue("backgroundColor") as? String ?? nil
```
call.get is to get the value from the plugin call, not from the `capacitor.config.json`

Reply:
OK, I have changed it according to your suggestion.